src/ConsoleMode.ps1

```diff
@@ -0,0 +1,103 @@
+# Hack! https://gist.github.com/lzybkr/f2059cb2ee8d0c13c65ab933b75e998c
+
+if ($IsWindows -eq $false) {
```

Comment:
For non-cross-platform PowerShell Core, we should test $PSVersionTable.PSVersion.Major -ge 6 -and $IsWindows.  As that variable is only defined in PowerShell Core.  We can assume a major version of -lt 6 is Windows only.